micromamba/tests/test_create.py

```diff
@@ -661,6 +666,39 @@ def test_create_envs_dirs(tmp_root_prefix: Path, tmp_path: Path):
     assert (tmp_path / env_name / "conda-meta" / "history").exists()
 
 
+@pytest.mark.parametrize("cli_root_prefix", (False, True))
+@pytest.mark.parametrize("user_envs_dirs", (False, True))
+def test_root_prefix_precedence(monkeypatch, tmp_path, user_envs_dirs, cli_root_prefix):
+    """Test for root prefix precedence"""
+
+    envroot = tmp_path / "envroot"
+    cliroot = tmp_path / "cliroot"
+    userenv = tmp_path / "userenv" / "envs"
+
+    map(lambda p: os.makedirs(p), (envroot, cliroot, userenv))
```

Comment:
Is this required because of https://github.com/mamba-org/mamba/issues/3790?

```suggestion
    map(lambda p: os.makedirs(p, exist_ok=True), (envroot, cliroot, userenv))
```

Reply:
Yes